ninja.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external ninja 1.8.2
+### RPM external ninja 1.10.0
```

Comment:
@forthommel , is `cepgen` really requires new ninja? If not then I would suggest to not change it here. This change will unnecessarily rebuild packages which depend on `ninja`

Reply:
Yes, I am afraid. Ninja 1.10 is required to build all Fortran dependencies (it was identified in the 14_1_X PR).

Reply:
Possibly if this is introducing too much noise in this build, we can switch to a standard Makefile build...

Reply:
I would like to avoid any rebuilds of existing packages for 10.6.X release cycle. So if using gmake works then please go ahead and update cepgen to use gmake

Reply:
note that updating `ninja` rebuilds [these packages](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-88804b/38039/externals/) . So if we can avoid these by using gmake then that will be very sefe